Removal of transformation scripts

mereolog opened the pull request:
This pull request removes the transformation scripts from the public repository.

aliariff commented:
@mereolog and what is the plan here? do we move it somewhere else? or these transformations are not required anymore?

mereolog commented:
> @mereolog and what is the plan here? do we move it somewhere else? or these transformations are not required anymore?

This PR is done at the request of the PA.
I moved the scripts to https://github.com/edmcouncil/idmp-transformations - please check whether I moved them all.

aliariff commented on .github/workflows/unit_tests_run.yml:
we can also remove this part
https://github.com/edmcouncil/idmp/pull/641/files#diff-f52a0b5af97cf0553b8b172795c6cc3720e033c78b3a5adecf9bd347c098a57cR9-R16

```
 workflow_dispatch:
    inputs:
      gsrs_version:
        description: "GSRS dataset version"
        required: false
      gsrs_cache_rebuild:
        description: "Whether to rebuild cache"
        required: false
```

Heiner-Oberkampf reviewed (approved):
Agreed with transition to non-public as decided by PA
